tests/e2escenarios/e2e_test.go

```diff
@@ -28,15 +27,41 @@ var _ = Describe("E2E Test", func() {
 
 	checkIfDevEnvIsUp := func(url, assertString string) {
 		Eventually(func() string {
-			resp, err := http.Get(fmt.Sprintf("http://%s", url))
-			Expect(err).ToNot(HaveOccurred())
+			resp, _ := http.Get(fmt.Sprintf("http://%s", url))
+			//nolint:govet
```

Comment:
Per the documentation of `http.Get`, if a non-nil error is returned, the `*Response`  returned along with it might be nil, so using it later might panic, I think.

Can you instead check the error and return maybe a zero-value string, e.g.:

```go
resp, err := http.Get(fmt.Sprintf("http://%s", url))
if err != nil {
  fmt.Fprintf(GinkgoWriter, "error while trying to GET %q: %v\n", url, err)
  return ""
}

```